Add info about ContentProviderLookupError (#1101)
plone/documentation: develop/plone/views/viewlets.rst

  <div tal:replace="structure provider:viewletmanagerid" />

> .. note ::

Reviewer: Spaces have meaning in reST. Minor syntax fix.

```suggestion
.. note::
```

Author: Done!

>     If you get a ``ContentProviderLookupError: viewletmanagerid`` you're trying to render a Plone page frame in an context which has not acquisition chain properly set up. Check `exceptions documentation <https://docs.plone.org/manage/troubleshooting/exceptions.html#contentproviderlookuperror-plone-htmlhead>`_ for more details.

Reviewer 2: No native speaker as well, but at least I found those:
```suggestion
    If you get a ``ContentProviderLookupError: viewletmanagerid`` you are trying to render a Plone page frame in a context which has no acquisition chain properly set up. Check `exceptions documentation <https://docs.plone.org/manage/troubleshooting/exceptions.html#contentproviderlookuperror-plone-htmlhead>`_ for more details.
```

Author: Sorry about that. It's just copy-pasted from the docs itself from https://docs.plone.org/manage/troubleshooting/exceptions.html#contentproviderlookuperror-plone-htmlhead.

Author: Done!
